lib/spack/docs/basic_usage.rst

```diff
@@ -1196,9 +1041,41 @@ of module files:
        """Set up the compile and runtime environments for a package."""
        pass
 
-"""""""""""""""""
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Regenerating Module files
+^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
@adamjstewart It seems that the sections added here b215b19cae1eaf665dfbec4e20c36497c57dd182 got partially erased at some point.